### PR TITLE
Notice legends

### LINKED ISSUE #10228: Show notice in legends panel if legends settings are off

### PR DESCRIPTION
**Pull request by nobuti**
This PR fixes #10228.

Also it introduces hidden layer notices for data and analyses panels. In addition, I'm moving analyses panel views inside its own folder following the convention we have for all layer content views.

screen shot 2016-10-19 at 11 00 24

CR @xavijam

### CODE REVIEW

**Comment by nobuti** [comment]
@brittanymicek could you take a look at all the copies?

File: lib/assets/javascripts/cartodb3/editor/layers/layer-content-views/analyses/analyses-view.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
+var Backbone = require('backbone');
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb3/editor/layers/layer-content-views/analyses/analyses-view.js]: Moved from other place right?

Comment (nobuti) [inline lib/assets/javascripts/cartodb3/editor/layers/layer-content-views/analyses/analyses-view.js]: The view that is moved is the content view of this.

Comment (xavijam) [inline lib/assets/javascripts/cartodb3/editor/layers/layer-content-views/analyses/analyses-view.js]: :es:

**Comment by xavijam** [comment]
Go for it man!

**Comment by brittanymicek** [comment]
@nobuti 

Change "Analyses" to "Analysis"

Under "Legends Disabled"
"The legends setting is disabled. Changes made will not be shown until you enable legends."

**Comment by nobuti** [comment]
Frontend test passing, merging 🚀

**Pull request merged**